src/main/java/com/wavefront/sdk/common/application/HeartbeaterService.java

```diff
@@ -49,6 +49,9 @@ public HeartbeaterService(WavefrontMetricSender wavefrontMetricSender,
         put(SHARD_TAG_KEY, applicationTags.getShard() == null ? Constants.NULL_TAG_VAL :
             applicationTags.getShard());
         put(COMPONENT_TAG_KEY, component);
+        if (applicationTags.getCustomTags() != null) {
+          putAll(applicationTags.getCustomTags());
```

Comment:
Should we consider handling any conflicts between custom and app tags? lgtm otherwise.

Reply:
Hopefully, no one will abuse application tags + custom tags :)
Let's not handle it and keep it simple.